cmd/consolelogs/aws/logging.go

```diff
@@ -0,0 +1,8 @@
+package aws
+
+import (
+	"github.com/bombsimon/logrusr"
+	"github.com/sirupsen/logrus"
+)
+
+var log = logrusr.NewLogger(logrus.New())
```

Comment:
Why do we this again and again in each cmd package instead of having a re-usable package that does that?

Reply:
to be able to do `log.Info` ... I'm not opposed to changing the way we do it, but maybe we can do that in a separate PR?